build/gamemaster_entrypoint.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -x
```

Comment:
Remove?

Reply:
Nope, debug messages in build scripts are always good